packages/synapse-react-client/src/components/FullTextSearch/FullTextSearch.test.tsx

```diff
@@ -0,0 +1,188 @@
+import { render, screen } from '@testing-library/react'
```

Comment:
This file was not type checked, so I fixed type issues and moved it to be colocated with the component. Made a few changes, so git thinks this is new.